scripts/select_appointments.sql

```diff
@@ -1 +1,14 @@
 -- retrieve appointment information
+
+-- Question 4. Retrieve the names and appointment times for everyone seeing Dr. Hubbard. Retrieve the Doctors' names and appointment times for Marsha Wilcox.
+
+SELECT d.surname, ...
```

Comment:
We;re looking for info on the _patients_ of Dr. Hubbard, which means that these columns should probably be one from the `people` table.